### PR TITLE
Add to_series method

### PR DESCRIPTION
**Pull request by henriqueribeiro**
- [x] Tests added / passed
- [x] Passes `flake8 dask`

Adding method `to_series()` on dask.dataframe.Index class

### CODE REVIEW

File: dask/dataframe/core.py [hunk 1]
```diff
@@ -2311,6 +2311,11 @@ def shift(self, periods=1, freq=None):
             freq = meta.freq
         return maybe_shift_divisions(out, periods, freq=freq)
 
+    @derived_from(pd.Index)
+    def to_series(self):
+        return self.map_partitions(M.to_series,
+                                   meta=self._meta.to_series())
```
Comment (mrocklin) [inline dask/dataframe/core.py]: My guess is that if you leave off the `meta=` keyword here that it will probably do the right thing anyway.

Comment (henriqueribeiro) [reply dask/dataframe/core.py]: Yep, I tried that also and it worked.

**Comment by henriqueribeiro** [comment]
Good! Should I do something?

**Comment by mrocklin** [comment]
> Good! Should I do something?

If you want to remove the `meta=` keyword then you should do so and then push the change back up to this branch.  Otherwise I'll just merge now.

**Comment by henriqueribeiro** [comment]
According to the documentation:
> This may lead to unexpected results, so providing meta is recommended.
So, maybe we should leave like this :)

**Comment by mrocklin** [comment]
In this particular case you would have been fine.  But lets just merge this for now.

Thanks @henriqueribeiro !

**Pull request merged**